apps/comps/app/api/auth/login/route.ts

```diff
@@ -57,7 +57,7 @@ export async function POST(req: NextRequest) {
       wallet: siweMessage.address,
     });
 
-    response.cookies.set("wallet", siweMessage.address, {
+    response.cookies.set("wallet_address", siweMessage.address, {
```

Comment:
note: i changed this because our backend is setting `wallet` via https://github.com/recallnet/js-recall/blob/e4c22a03951ab3f1ec9ae32fbb0dfeaf15909901/apps/api/src/types/index.ts#L281

i dont have a strong preference and can clean it up once we connect the app to the api in #331

Reply:
makes sense, once we integrate with the backend we'll probably remove the local api